### PR TITLE
BTAT-3138: Added payment history helper

### PR DESCRIPTION
**Pull request by ja-green**

### CODE REVIEW

File: app/views/templates/paymentsHistoryCharge.scala.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  (firstDate, secondDate) match {
+    case (Some(fd), Some(sd)) => {
+      messages(
+        s"$chargeTypeDescription",
+        s"${fd.getDayOfMonth} ${fd.getMonth.toString.head}${fd.getMonth.toString.toLowerCase.substring(1, 3)}",
+        s"${sd.getDayOfMonth} ${sd.getMonth.toString.head}${sd.getMonth.toString.toLowerCase.substring(1, 3)} ${sd.getYear}"
+      )
+    }
+
+    case (Some(fd), None) => {
```
Comment (TomDudley92) [inline app/views/templates/paymentsHistoryCharge.scala.html]: To increase the coverage can you add tests for the first or second date being a None please. I'm not sure though that we can get a charge with only one date though so those cases may not be required
